.github/workflows/ci.yml

```
@@ -293,3 +293,11 @@ jobs:
       - name: Check image build
         shell: bash
         run: bash -x scripts/tests.build_image.sh
+  govulncheck:
+    runs-on: ubuntu-latest
+    name: Run govulncheck
```

This shows up as: `Tests / Run govulncheck (pull_request)`. Should we just name this `govulncheck`?
```suggestion
    name: govulncheck
```